static/ingredients.html

```diff
@@ -72,11 +71,26 @@
 
 
 <script>
+    $(function () {
+        getProductsRequest(products => {
+            products.forEach(pr =>
+                addProductToTable(pr.name, pr.kcal, pr.id)
+            );
+        });
+    });
+
     function addProductToTable(name, kcal, dbId) {
         let dltBtn = $("<input type=submit value=Delete>")
             .attr("dbId", dbId)
```

Comment:
you don't need to store this attrs on the button - you use them only in onClick function which captures these variables already.
So this code would do the same:
```js
$("<button>Delete</button>").click(() =>
deleteProductRequest(`${name}, ${kcal}`, dbId, ...)};
```

Same goes for other similar places where you create html elements from js.